src/Terminal/commands/ls.tsx

```diff
@@ -139,10 +139,12 @@ export function ls(
       }),
     )();
 
-    const rowSplit = row
-      .split(" ")
-      .map((x) => x.trim())
-      .filter((x) => !!x);
+    console.log(row);
+
+    const rowSplit = row.split("~");
+    let rowSplitArray = rowSplit.map((x) => [x.trim(), x.replace(x.trim(), "")]);
+    console.log(rowSplitArray);
+    rowSplitArray = rowSplitArray.filter((x) => !!x);
```

Comment:
When would this filter anything? `rowSplitArray`'s elements are two element arrays, so they are always truthy, aren't they?

Reply:
It was already in there, so I left it, but didn't change it to look at the first element of the array.